views/doc/index.handlebars

```diff
@@ -0,0 +1,18 @@
+<div class="row">
+  <div class="col-md-12">
+    {{{nav}}}
+    <p><a href="../">&uarr; up to Specberus home</a></p>
+    <h2>Specberus documentation</h2>
+
+    <h3 id="publication"><a href="#publication">Publication rules</a></h3>
+    <ul>
+      {{#each profiles}}
+        <li><a href="/doc/rules?profile={{this.abbr}}">{{this.name}} ({{this.abbr}})</a></li>
+      {{/each}}
```

Comment:
Seems like we are missing the submissions doc. We don't receive many submissions so we can keep that for later. I created a new ticket so we don't forget it: #369